### ISSUE TITLE
Removing BUILTIN_IMAGES

### ISSUE THREAD

**Issue by evadne** [AlanQuatermain/AQGridView#111]
Actually, that don’t even work at all with what we have.  Can we remove it?

**Comment by iprebeg** [comment]
hey, is this the reason why aqgridviewcell ignores gray and other selection styles besides glow?

**Comment by evadne** [comment]
@iprebeg yes.  In fact, since we’re no longer targeting IOS <5.0, the canonical way to do it may be using `-[CALayer setShadowColor:]` or subclass the cell and provide your own image file.
